locale/cups_de.po

```diff
@@ -4248,10 +4248,10 @@ msgid "Tabloid"
 msgstr "US Tabloid"
 
 msgid "Tabloid Oversize"
-msgstr ""
+msgstr "US Tabloid übergross"
 
 msgid "Tabloid Oversize Long Edge"
-msgstr ""
+msgstr "US Tabloid übergross lange Seite"
```

Comment:
should be "übergroß" not "übergross" if this is not considered to be Swiss german